BackofficeBundle/Security/Authorization/Voter/NodeGroupRoleVoter.php

```diff
@@ -90,4 +98,28 @@ public function vote(TokenInterface $token, $object, array $attributes)
 
         return self::ACCESS_ABSTAIN;
     }
+
+    /**
+     * @param NodeInterface  $node
+     * @param GroupInterface $group
+     * @param string         $attribute
+     *
+     * @return boolean
+     */
+    protected function isGrantedNodeGroupRole(NodeInterface $node, GroupInterface $group, $attribute)
+    {
+        $nodeGroupRole = $group->getNodeRoleByNodeAndRole($node->getNodeId(), $attribute);
+        if ($nodeGroupRole instanceof NodeGroupRoleInterface) {
+            if ($nodeGroupRole->isGranted() === 'inherit') {
```

Comment:
the isser should return only a boolean, so either we change from `isGranted` to `getGranted` or to `getGrantedType`